redisinsight/api/src/modules/database/providers/database-info.provider.ts

```diff
@@ -261,4 +258,54 @@ export class DatabaseInfoProvider {
       throw catchAclError(e);
     }
   }
+
+  public async getRedisInfo(client: RedisClient) {
+    try {
+      return convertRedisInfoReplyToObject(
+        (await client.sendCommand(['info'], {
+          replyEncoding: 'utf8',
+        })) as string,
+      );
+    } catch (error) {
+      if (error.message.includes(ERROR_MESSAGES.NO_INFO_COMMAND_PERMISSION)) {
+        // Fallback to getting basic information from `hello` command
+        return this.getRedisHelloInfo(client);
+      }
+
+      throw error;
```

Comment:
what will happen when `hello` command is also disabled?
Also you are handling an error in different way for `info` and `hello` commands. Is it expected behavior or we need to add `catchAclError` here too?